docs/configuration.md

```diff
@@ -485,7 +526,7 @@ creating trades on the exchange.
 
 ```json
 "exchange": {
-        "name": "bittrex",
+    "name": "bittrex",
```

Comment:
I'm not per se against this - but if we change formatting of this line - then please also for the others ...